src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -188,40 +198,56 @@ export default {
                 image.selected = true;
             }
         },
-        stopListening() {
-            if (this.unsubscribe) {
-                this.unsubscribe();
-                delete this.unsubscribe;
+        boundsChange(bounds, isTick) {
+            this.bounds = bounds;
+            this.requestHistory(isTick);
```

Comment:
Let's not call it at all if `isTick` is true.

Reply:
Updated